packages/@aws-cdk/aws-codepipeline-actions/lib/codebuild/build-action.ts

```diff
@@ -176,6 +183,9 @@ export class CodeBuildAction extends Action {
       // lazy, because the Artifact name might be generated lazily
       configuration.PrimarySource = cdk.Lazy.string({ produce: () => this.props.input.artifactName });
     }
+    if (this.props.batchEnabled) {
+      configuration.BatchEnabled = 'true';
```

Comment:
Are we 100% certain this should be a String, and not a Boolean?

Reply:
In the example on https://docs.aws.amazon.com/codebuild/latest/userguide/sample-pipeline-batch.html it's a string, which is a bit strange given it's JSON 🤷‍♂️